packages/components/stories/List.js

```diff
@@ -162,59 +162,68 @@ const props = {
 	list: {
 		columns: [
 			{ key: 'id', label: 'Id', order: 0 },
-			{ key: 'name', label: 'Name', order: 1 },
-			{ key: 'author', label: 'Author', order: 3 },
-			{ key: 'created', label: 'Created', order: 2 },
+			{ key: 'name', selector: 'meta.name', label: 'Name', order: 1 },
+			{ key: 'author', selector: 'meta.author', label: 'Author', order: 3 },
+			{ key: 'created', selector: 'meta.created', label: 'Created', order: 2 },
```

Comment:
I think devs that are used to simple data without selector can be confused if you pass almost all the example with a selector.
IMO you should only pass 1 column (or a new one) into nested data with selector. Or perhaps, if you want to have coherent data, you can group the created/modified date into a nested `meta` property. That would be 2 columns with selector.